app/models.py

```diff
@@ -14,17 +14,26 @@ class Recruiter(models.Model):
     first_name = models.CharField(max_length=50)
     last_name = models.CharField(max_length=50)
     email = models.CharField(max_length=50)
-    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    organization = models.CharField(max_length=50)
     is_superadmin = models.BooleanField(default=False)
 
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
+    if created:
+        Recruiter.objects.create(user=instance)
+
+@receiver(post_save, sender=User)
+def save_user_profile(sender, instance, **kwargs):
```

Comment:
Tutaj w skrócie to co robisz to po tym jak zapiszesz użytkownika to zapisujesz dodatkowo (bez zmian) rekrutera. To tak nie powinno wyglądać. W skrócie - to nic nie robi